### ISSUE TITLE
[Feature request]: Double pipe ligature (logical OR operator in JavaScript)

### ISSUE THREAD

**Issue by m8ee7a** [MonoLisaFont/feedback#264]
### What feature are you proposing?

Love the font, but I noticed that it's lacking a ligature for the logical "or" operator in JavaScript.

Cheers!

**Comment by bebraw** [comment]
Do you mean just a spacing fix or something else? It would help if you can send some visual examples of what might work for you.

**Comment by m8ee7a** [comment]
Hi there @bebraw! I think it's more of a spacing fix? Basically when I use an `OR` (`||`) statement in my code, the two pipes don't appear as a single ligature. Here's an example of the difference between Fira Mono and Fira Code that I hope helps to convey what I mean:

![double-pipe](https://github.com/MonoLisaFont/feedback/assets/165081140/1b36603e-6c67-4978-874e-6da3472af302)

Thanks again for the fantastic font :)

**Comment by BlueskyFR** [comment]
I agree! Would be great to match the number of ligatures already present in Fira Code tbh
